tests/publiccloud/instance_overview.pm

```diff
@@ -55,11 +55,23 @@ sub run {
         zypper_call("in bzip2");
     }
 
-    assert_script_run("rpm -qa > /tmp/rpm.list.txt");
-    upload_logs('/tmp/rpm.list.txt', timeout => 180, failok => 1);
-
     assert_script_run("SUSEConnect --status-text", 300);
     zypper_call("lr -d");
+
+    collect_system_information();
+}
+
+sub collect_system_information {
+    # Collect various system information and pack them to instance_overview.tar
+    script_run("cd /var/tmp");
+    assert_script_run("mkdir -p instance_overview");
+    assert_script_run("rpm -qa | tee instance_overview/rpm.list.txt");
+    assert_script_run("cat /proc/cpuinfo | tee instance_overview/cpuinfo.txt");
+    assert_script_run("cat /proc/meminfo | tee instance_overview/meminfo.txt");
+    assert_script_run("uname -a | tee instance_overview/uname.txt");
+    assert_script_run("tar -cf instance_overview.tar instance_overview");
+    upload_logs('instance_overview.tar', timeout => 180, failok => 1);
```

Comment:
```suggestion
    assert_script_run("tar -xcf instance_overview.tar.gz instance_overview/");
    upload_logs('instance_overview.tar.gz', timeout => 180, failok => 1);
```

Reply:
ok

Reply:
I'm sorry, I messed the parameters up :( It's `tar czf` not `tar xcf`.

Reply:
... and I was probably too tired to notice it as well :sweat_smile: 
Fixing it right now